docs/using/administering/debugging.rst

```diff
@@ -33,6 +37,78 @@ It is possible to see the available unit names, and then view the logs with ``jo
    [root@node1]# journalctl -u flocker-container-agent
    [root@node1]# journalctl -u flocker-control
 
+Bug Reporting
+-------------
+
+When reporting issues with Flocker please include:
+
+* Flocker version,
+* operating system and version,
+* Linux kernel version,
+* Docker version and configuration,
+* node IP addresses,
+* node hostname,
+* recent syslog content,
+* and separate Flocker log files.
```

Comment:
:question: In our recent conversations when helping some users, we've also requested the following outputs:

* `cinder list`
* `nova volume-list`
* `lsblk -a`
* `fdisk -l`

I would definitely recommend **not** doing the openstack stuff as part of this PR.

I am less sure about whether we should include the other two. On the one hand, we know we need them, on the other hand, this PR is already a huge improvement on the current situation, so I'm inclined to get it "in the bank", so to speak.

If you do add `lsblk -a` and `fdisk -l`, please resubmit for further review.